### PR TITLE
Fixed compact.js added convertInchesToFeet.js

### PR DESCRIPTION
**Pull request by Manny1995**
Closes #theRelatedIssueNumber

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

**Comment by mtso** [comment]
Closes #376

File: lib/compact.js [hunk 1]
```diff
@@ -6,8 +6,36 @@
...
+  }
+
+  var newArray = new Array();
+  arr.forEach(function(val) {
+    if (isFalsy(val) == false){
+      newArray.push(val);
+    }
+  });
+
+  return newArray;
```
Comment (ktilcu) [inline lib/compact.js]: We really appreciate your contribution and look forward to getting this merged but I think we should look at the implementation.  Compact can be a really simple function with the right tools. Look into `Array.prototype.filter` and the identity function. I think you'll be surprised.

**Comment by Manny1995** [comment]
@ktilcu thanks for the heads up, that filter and identity really surprised me!  Made the changes, and going to create a new pull request.

**Comment by ktilcu** [comment]
@Manny1995 Thanks for the work! Glad the tip panned out. As another tip, `identity` is an important function for any library. If you make a new PR breaking that function out with some tests I will get it merged. In the meantime, I will merge this PR. If you could make an issue for your new skeleton method I would be grateful. 👍

**Pull request merged**